airqo-platform/AirQo-frontend
# [Netmanager] User Management Features

- What does this PR do?
Incorporates many components for user management into the Netmanager product - transfers them from "Analytics"
Same PR also updates the favicon of the product - as seen in the browser window.
It also disables the registration page - leaving the Login page as the only option.

- How to test out PR?
`cd netmanager`
`npm install`

> For MAC

`npm run dev-mac` for development environment
`npm run prod-mac` for production environment
`npm run stage-mac` for staging environment

> For Windows

`npm run dev-pc` for development environment
`npm run prod-pc` for production environment
`npm run stage-pc` for staging environment

- Any considerations?
When running in the **development environment**, please ensure that the **auth-service microservice** is running at the same time.
Otherwise, no need to do if you are running in the production environment.
